app/experimenter/experiments/models/nimbus.py

```diff
@@ -297,6 +299,18 @@ def __str__(self):  # pragma: no cover
         return self.name
 
 
+class NimbusExperimentProbeSets(models.Model):
+    experiment = models.ForeignKey(
+        NimbusExperiment,
+        on_delete=models.CASCADE,
+    )
+    probe_set = models.ForeignKey(NimbusProbeSet, on_delete=models.CASCADE)
+    is_primary = models.BooleanField(null=False)
```

Comment:
Shouldn't this have `default=False`?

Reply:
I considered that, but didn't want errors to occur if someone forgot to specify what it should be. I can't think of any good reason off-hand why it should default to one or the other.

Reply:
Seems reasonable.